complex/ft_config.c

```diff
@@ -669,7 +669,7 @@ void fts_cur_info(struct ft_series *series, struct ft_info *info)
 	info->class_function = set->class_function[series->cur_func];
 	info->op = set->op[series->cur_op];
 	info->test_flags = set->test_flags;
-	info->caps = set->caps[series->cur_caps];
+	info->caps = set->caps[series->cur_caps] | FT_CAP_MSG;
```

Comment:
Since the RMA and atomic testing both send messages as part of their testing, wouldn't we still need to request MSG capabilities? Or should we try to be using sockets to coordinate those writes?